src/silo/storage/sequence_store.cpp

```diff
@@ -135,10 +138,61 @@ void silo::SequenceStorePartition::fillNBitmaps(const std::vector<std::string>&
    });
 }
 
+void silo::SequenceStorePartition::fillMutationFilter() {
+   const auto start_length = 8;
+   std::vector<uint32_t> slice_lengths;
+
+   for (uint32_t slice_length = start_length; slice_length <= reference_genome.size();
+        slice_length *= 2) {
+      slice_lengths.push_back(slice_length);
+   }
+
+   std::mutex mutex;
+   tbb::parallel_for_each(slice_lengths.begin(), slice_lengths.end(), [&](const auto slice_length) {
+      const auto overlap_shift = slice_length / 2;
+      for (uint32_t div = start_length; div >= 1; div /= 2) {
+         const auto mutation_count = slice_length / div;
+         std::vector<std::unique_ptr<roaring::Roaring>> bitmaps;
+         for (uint16_t idx = 0; idx < reference_genome.size(); idx += overlap_shift) {
+            std::vector<uint32_t> counts(sequence_count, 0);
+            const auto end =
+               std::min(static_cast<size_t>(idx + slice_length), reference_genome.size());
+            for (uint16_t genome_pos = idx; genome_pos < end; ++genome_pos) {
+               const auto* negated_reference_bitmap =
+                  this->getBitmap(genome_pos, this->reference_genome[genome_pos]);
+               const auto* n_symbol_bitmap = this->getBitmap(genome_pos, NUCLEOTIDE_SYMBOL::N);
+               auto pure_mutation = std::make_unique<roaring::Roaring>(roaring_bitmap_andnot(
+                  &negated_reference_bitmap->roaring, &n_symbol_bitmap->roaring
+               ));
```

Comment:
@Taepper I did not find where the reference genome bitmap gets flipped. Does SILO still do it? Anyways, this part is not yet correct. Should we do a union over all bitmaps except the reference bitmap to get the ones that are actually mutated to a symbol other than N